cmd/pod.go

```diff
@@ -67,6 +75,17 @@ func (p *podHandler) OnDelete(obj interface{}) {
 	}
 }
 
+func keysDiffer(oldAnnotations, newAnnotations map[string]string, keysThatMatter []string) bool {
```

Comment:
Just trying to make it usable if there were additional annotations that were used in the future, not specific reason.  I'll rename and change.